stories/features/rewards/concepts.tsx

```diff
@@ -129,5 +139,137 @@ storiesOf('Feature Components/Rewards/Concepts', module)
+
+    const getProp = (show: boolean, prop: string): any => {
+      switch (prop) {
+        case 'opacity':
+          return show ? 1 : 0
+          break
+        case 'overflow':
+          return show ? 'unset' : 'hidden'
+          break
+        case 'height':
+          return show ? '100%' : 0
+        default:
+          return ''
+      }
+    }
+
+    return (
+      <div style={{ background: `url(${tipScreen}) no-repeat top center`, width: '986px', height: '912px', margin: '0 auto', position: 'relative' }}>
+        <div style={{ position: 'absolute', bottom: '185px', left: '330px' }}>
+          <WalletWrapper
+            contentPadding={!store.state.showPanel}
+            tokens={number('Tokens', 25)}
+            converted={text('Converted', '163230.50 USD')}
+            actions={[
+              {
+                name: 'Add funds',
+                action: doNothing,
+                icon: wallet
+              },
+              {
+                name: 'Withdraw Funds',
+                action: doNothing,
+                icon: funds
+              }
+            ]}
+            showCopy={boolean('Show Uphold', false)}
+            showSecActions={boolean('Show secondary actions', true)}
+            connectedWallet={boolean('Connected wallet', false)}
+            grants={object('Grants', [
+              {
+                tokens: 8,
+                expireDate: '7/15/2018'
+              },
+              {
+                tokens: 10,
+                expireDate: '9/10/2018'
+              },
+              {
+                tokens: 10,
+                expireDate: '10/10/2018'
+              }
+            ])}
+          >
+            <div
+              style={{
+                transition: 'visibility 0s, opacity 0.5s linear',
+                overflow: getProp(store.state.showPanel, 'overflow'),
+                opacity: getProp(store.state.showPanel, 'opacity'),
+                height: getProp(store.state.showPanel, 'height')
+              }}
+            >
+              <WalletPanel
+                id={'wallet-panel'}
+                platform={'youtube'}
+                publisherImg={bartBaker}
+                publisherName={'Bart Baker'}
+                monthlyAmount={10}
+                isVerified={true}
+                tipsEnabled={boolean('Tips enabled', store.state.tipsEnabled)}
+                includeInAuto={boolean('Tips enabled', store.state.includeInAuto)}
+                attentionScore={'17'}
+                donationAmounts={
+                  [5, 10, 15, 20, 30, 50, 100]
+                }
+                onToggleTips={onToggleTips}
+                donationAction={doNothing}
+                onAmountChange={doNothing}
+                onIncludeInAuto={onIncludeInAuto}
+              />
+              <WalletSummarySlider
+                title={true}
+                id={'panel-slider'}
+                onToggle={onToggleSummary}
+              />
+            </div>
+            <div
```

Comment:
Transition should be part of `WalletSummarySlider`

Reply:
The transition (from visible -> hidden) occurs on the <WalletPanel> component. Not sure how this would work contained in <WalletSummarySlider>

Reply:
So my idea was that we have both profile screen and summary screen inside `WalletSummarySlider`. In there we can create this wrapper transition divs.

Reply:
I was going to do it like this:

`<WalletSummarySlider>
   <WalletPanel {...props}/>
   <WalletSummary {...props}/>
 </WalletSummarySlider>`

Then I can enumerate through the children, generate a wrapper div around each, and handle the transition

Reply:
would love to see it implemented like this